### PR TITLE
Delegate screen

### PR DESCRIPTION
**Pull request by georgeburduhos**

### CODE REVIEW

File: src/screens/pos-actions/pos-action-delegate-select-validator/pos-action-delegate-select-validator.tsx [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+                            blockchainInstance
+                                .getStats(this.props.chainId)
+                                .getValidatorList(CardActionType.NAVIGATE, nrValidatorsNew)
+                                .then(validators => {
+                                    this.setState({
+                                        nrValidators: nrValidatorsNew,
+                                        validatorsList: validators
+                                    });
+                                })
+                                .catch();
```
Comment (NovacAndrei) [inline src/screens/pos-actions/pos-action-delegate-select-validator/pos-action-delegate-select-validator.tsx:138]: Just a reminder, if you consider this error from catch to be useful, we can catch this error to sentry.

File: src/screens/token/components/delegate-token/components/validators/validators-list/validators-list.tsx [hunk 2]
```diff
@@ -1,37 +1,48 @@
...
 
 export const ValidatorsListComponent = (
     props: IExternalProps & IThemeProps<ReturnType<typeof stylesProvider>>
 ) => {
+    const config = getBlockchain(props.blockchain).config;
+
     return (
-        <View style={props.styles.container}>
-            {props.validators.map((validator: IValidatorCard, index: number) => (
+        <ScrollView contentContainerStyle={props.styles.container}>
```
Comment (NovacAndrei) [inline src/screens/token/components/delegate-token/components/validators/validators-list/validators-list.tsx:26]: Please also add here: `showsVerticalScrollIndicator={false}` when using ScrollView

File: src/navigation/navigation.tsx [hunk 3]
```diff
@@ -144,6 +155,41 @@ export const WalletNavigation = createStackNavigator(
         Validator: {
             screen: ValidatorScreen
         },
+        DelegateSelectValidator: {
```
Comment (krisboit) [inline src/navigation/navigation.tsx:158]: first screen from flows should be renamed. For this specific case: `PosDelegate`. Also all POS screen should be prefixed with `Pos`

File: src/screens/pos-actions/pos-action-quick-delegate/quick-delegate-confirm/quick-delegate-confirm.tsx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+    }
+
+    private async onPressConfirm() {
+        try {
+            await PasswordModal.getPassword(
+                translate('Password.pinTitleUnlock'),
+                translate('Password.subtitleSignTransaction'),
+                { sensitive: true, showCloseButton: true }
+            );
+            NavigationService.goBack('TokenScreen-key');
```
Comment (NovacAndrei) [inline src/screens/pos-actions/pos-action-quick-delegate/quick-delegate-confirm/quick-delegate-confirm.tsx:107]: Is this working? If not, left a TODO not to forget about this.

File: src/screens/pos-actions/components/validator-delegate-amount/styles.ts [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        },
+        labelContainer: {
+            flexDirection: 'row',
+            alignItems: 'baseline',
+            marginBottom: BASE_DIMENSION / 2
+        },
+        labelName: {
+            fontSize: normalizeFontAndLineHeight(16),
+            fontWeight: '500',
+            letterSpacing: 0.39,
```
Comment (NovacAndrei) [inline src/screens/pos-actions/components/validator-delegate-amount/styles.ts:32]: Here please use:
```
letterSpacing: LETTER_SPACING,
```

File: src/screens/pos-actions/components/validator-delegate-amount/styles.ts [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        },
+        rankText: {
+            fontSize: normalizeFontAndLineHeight(10),
+            color: theme.colors.textTertiary
+        },
+        amount: {
+            fontSize: normalizeFontAndLineHeight(14),
+            lineHeight: normalize(19),
+            fontWeight: '500',
+            letterSpacing: 0.38,
```
Comment (NovacAndrei) [inline src/screens/pos-actions/components/validator-delegate-amount/styles.ts:49]: Same here

File: src/screens/pos-actions/pos-action-quick-delegate/quick-delegate-select-validator/quick-delegate-select-validator.tsx [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+                            .then(validators => {
+                                this.setState({
+                                    nrValidators: nrValidatorsNew,
+                                    validatorsList: validators
+                                });
+                            })
+                            .catch();
+                    }}
+                >
+                    <Icon name="plus" size={normalize(16)} style={styles.actionIcon} />
```
Comment (NovacAndrei) [inline src/screens/pos-actions/pos-action-quick-delegate/quick-delegate-select-validator/quick-delegate-select-validator.tsx:148]: Please use here: 
```
name={IconValues.PLUS}
```

File: src/screens/pos-actions/pos-action-quick-delegate/quick-delegate-select-validator/quick-delegate-select-validator.tsx [hunk 8]
```diff
@@ -0,0 +1,229 @@
...
+                                        nrValidators: nrValidatorsNew,
+                                        validatorsList: validators
+                                    });
+                                })
+                                .catch();
+                        }
+                        // decrease
+                    }}
+                >
+                    <Icon name="plus" size={normalize(16)} style={styles.actionIcon} />
```
Comment (NovacAndrei) [inline src/screens/pos-actions/pos-action-quick-delegate/quick-delegate-select-validator/quick-delegate-select-validator.tsx:129]: Please user here:

```
name={IconValues.PLUS}
```

File: src/screens/pos-actions/pos-action-redelegate/redelegate-select-validator/redelegate-select-validator.tsx [hunk 9]
```diff
@@ -0,0 +1,275 @@
...
+                                        nrValidators: nrValidatorsNew,
+                                        validatorsList: validators
+                                    });
+                                })
+                                .catch();
+                        }
+                        // decrease
+                    }}
+                >
+                    <Icon name="plus" size={normalize(16)} style={styles.actionIcon} />
```
Comment (NovacAndrei) [inline src/screens/pos-actions/pos-action-redelegate/redelegate-select-validator/redelegate-select-validator.tsx:166]: Same here

File: src/screens/pos-actions/pos-action-redelegate/redelegate-select-validator/redelegate-select-validator.tsx [hunk 10]
```diff
@@ -0,0 +1,275 @@
...
+                            .then(validators => {
+                                this.setState({
+                                    nrValidators: nrValidatorsNew,
+                                    validatorsList: validators
+                                });
+                            })
+                            .catch();
+                    }}
+                >
+                    <Icon name="plus" size={normalize(16)} style={styles.actionIcon} />
```
Comment (NovacAndrei) [inline src/screens/pos-actions/pos-action-redelegate/redelegate-select-validator/redelegate-select-validator.tsx:185]: Same here

File: src/screens/pos-actions/components/validator-delegate-amount/styles.ts [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+            fontSize: normalizeFontAndLineHeight(11),
+            color: theme.colors.textSecondary
+        },
+        rankText: {
+            fontSize: normalizeFontAndLineHeight(10),
+            color: theme.colors.textTertiary
+        },
+        amount: {
+            fontSize: normalizeFontAndLineHeight(14),
+            lineHeight: normalize(19),
```
Comment (NovacAndrei) [inline src/screens/pos-actions/components/validator-delegate-amount/styles.ts:48]: Please use here `normalizeFontAndLineHeight(19)`

File: src/styles/themes/dark-theme.ts [hunk 12]
```diff
@@ -34,6 +34,10 @@ export const darkTheme: ITheme = {
         disabledButton: COLORS.DARK_GRAY,
 
         gradientLight: COLORS.COD_GRAY,
-        gradientDark: COLORS.BLACK
+        gradientDark: COLORS.BLACK,
+
+        undelegate: COLORS.ORANGE,
```
Comment (krisboit) [inline src/styles/themes/dark-theme.ts:39]: these are too generic, use a more detailed name, something like `labelUndelegate`

**Pull request merged**